api/db/changelog/db.changelog-45-concept-set.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="danrodney" id="changelog-45">
+
+    <createTable tableName="concept_set">
+      <column name="concept_set_id" type="bigint" autoIncrement="true">
+        <constraints primaryKey="true" nullable="false"/>
+      </column>
+      <column name="workspace_id" type="bigint">
+        <constraints nullable="false"/>
+      </column>
+      <column name="name" type="varchar(80)">
+        <constraints nullable="false"/>
+      </column>
+      <column name="description" type="clob">
+        <constraints nullable="true"/>
+      </column>
+      <column name="domain" type="tinyint">
```

Comment:
So a concept set is homogeneous for a single domain - that sounds reasonable to me though I don't know off-hand whether or not that's a hard requirement, perhaps worth adding a sentence with rationale in your design doc.

Reply:
The idea is a single concept set can be used to retrieve data from a single OMOP table during cohort materialization, so it's important that all the concepts are from the same domain. I will update the doc.